src/ng2-auto-complete.directive.ts

```diff
@@ -46,6 +47,7 @@ export class Ng2AutoCompleteDirective implements OnInit {
   inputEl: HTMLInputElement;  // input tag
 
   constructor(private resolver: ComponentFactoryResolver,
+              private renderer: Renderer,
               public  viewContainerRef: ViewContainerRef) {
     this.el = this.viewContainerRef.element.nativeElement;
   }
```

Comment:
The same thing, let's keep two space indentation rather than having ruby-on-rails like coding style.